source/_patterns/04-pages/research-article/research-article~en.yaml

```diff
@@ -23,6 +23,7 @@ contentHeader:
           aria-label: Issue number
         content:
           text: Issue 1
+    date: now
```

Comment:
This needs move to the correct place. (We should use `2018-12-18`, and add it to the other languages too.)